matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/auth/version/Versions.kt

```diff
@@ -153,3 +155,13 @@ private fun Versions.getMaxVersion(): HomeServerVersion {
 internal fun Versions.doesServerSupportRemoteToggleOfPushNotifications(): Boolean {
     return unstableFeatures?.get(FEATURE_REMOTE_TOGGLE_PUSH_NOTIFICATIONS_MSC3881).orFalse()
 }
+
+/**
+ * Indicate if the server supports MSC3912: https://github.com/matrix-org/matrix-spec-proposals/pull/3912.
+ *
+ * @return true if event redaction with relations is supported
+ */
+internal fun Versions.doesServerSupportRedactEventWithRelations(): Boolean {
+    return unstableFeatures?.get(FEATURE_EVENT_REDACTION_WITH_RELATIONS).orFalse() ||
+            unstableFeatures?.get(FEATURE_EVENT_REDACTION_WITH_RELATIONS_STABLE).orFalse()
```

Comment:
When the MSC lands into the spec, `.stable` is added at the end of the key (from what I understood), there are some examples [here](https://matrix.org/_matrix/client/versions). According to this key & if the feature is enabled, I should use the stable or unstable key in the redaction query but I could not find a way to do it currently excepted by sending both because we only store in the homeserver capabilities if the feature is enabled or not.

Reply:
In my opinion, we should keep the verification for both keys (otherwise it will not work anymore when the feature will become stable). But I think we can only send the unstable field in the request. The hs should be able to handle it for a certain time whether the feature is stable or not, wdyt @bmarty / @mnaturel ?

Reply:
Agree to keep these fields, I was talking about not using [this one](https://github.com/vector-im/element-android/blob/dd9fba062688085aee8dd9ea987e7e0fab2ed809/matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/send/model/EventRedactBody.kt#L27-L28) until the MSC is not landed

Reply:
Hum yes I think we should only send the unstable field for now in the body.

Reply:
Done https://github.com/vector-im/element-android/pull/7988/commits/0870cec118152665f95741f70680e885dae33792